integration-tests/cli/customer-erasure.it.js

```diff
@@ -0,0 +1,153 @@
+import CustomerErasure from '@commercetools/customer-erasure'
+import tmp from 'tmp'
+import { getCredentials } from '@commercetools/get-credentials'
+import { exec } from 'mz/child_process'
+import { version } from '@commercetools/customer-erasure/package.json'
+import { createData, clearData, getId } from './helpers/utils'
+import {
+  customer,
+  order,
+  cart,
+  payment,
+  shoppingList,
+  review,
+  customLineItem,
+} from './helpers/customer-erasure.data'
+
+let projectKey
+if (process.env.CI === 'true') projectKey = 'custom-objects-import-int-tests'
+else projectKey = process.env.npm_config_projectkey
+
+describe('customer erasure', () => {
+  const bin = './integration-tests/node_modules/.bin/customer-erasure'
```

Comment:
Nitpick: `yarn run customer-erause` or `$(npm bin)/customer-erasure`

Reply:
Above didnt work out for me when running locally so I kept the path like this for now.